masterlist.yaml

```
@@ -2084,6 +2084,15 @@ plugins:
       - *alreadyInOrFixedByFCF
       - *alreadyInOrFixedByPRPv0_59
 
+  - name: 'Better Landscape Grass.esp'
+    url: ["https://www.nexusmods.com/fallout4/mods/38360/" ]
+    group: *lowPriorityGroup
+    dirty:
+      - <<: *quickClean
+        crc: 0x7E2E53AF
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 191
+        
   - name: 'CliffsEdgeHotelWindowFix.esp'
```

There are some trailing spaces here:

```suggestion
        itm: 191

  - name: 'CliffsEdgeHotelWindowFix.esp'
```